### ISSUE TITLE
Timetracker export to ODS contains other fields then to CSV

### ISSUE THREAD

**Issue by gho59** [tine20/tine20#6922]
Version Tine 2018.08.4 on Ubuntu 18.04 (repo: https://packages.tine20.org/ubuntu/ bionic)
The Timetrackerexport to ODS contains not all visible fields from listing. Export to CSV is working like expected.
Will this be fixed in near future?

greets gho59

thanks for the good stuff

**Comment by pschuele** [comment]
hey there,
if you like, you can help us by adding the missing fields here: https://github.com/tine20/tine20/blob/master/tine20/Timetracker/Export/definitions/ts_default_ods.xml
just run 
```
php --config /etc/tine20/config.inc.php setup.php --updateAllImportExportDefinitions
```
to update the definition in tine20.

**Comment by gho59** [comment]
I created a new file "ts_default_ods_neu.xml" with all Fields from table "x_timetracker_timesheet".
It would be nice to get the field "Title" instead of "timeaccount_id" , but I wasn't able to find out how.

**Comment by gho59** [comment]
It would also be nice to get the list of definitionfiles in the dropdownlist for "Export as...". At the moment there is just the "ts_default_ods.xml" as a choice.
Greets gho59

**Comment by gho59** [comment]
Got it done to give to github and create a pullrequest. editing wasn't allowed for me.

**Comment by pschuele** [comment]
PR is here: https://github.com/tine20/tine20/pull/6932

**Comment by pschuele** [comment]
you could also add this to your commit message (in the PR) to close this issue:

```
fixes #6922
```
